src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml

```
@@ -9,21 +9,62 @@ sylius_fixtures:
                 logger: ~
 
             fixtures:
-                locale: ~
+                locale:
+                    options:
+                        locales:
+                            - 'en_US'
+                            - 'de_DE'
+                            - 'fr_FR'
+                            - 'pl_PL'
+                            - 'es_ES'
+                            - 'es_MX'
+                            - 'pt_PT'
+                            - 'zh_CN'
                 currency:
                     options:
                         currencies:
                             - 'USD'
+                            - 'PLN'
+                            - 'CAD'
+                            - 'CNY'
+                            - 'NZD'
+                            - 'GBP'
+                            - 'AUD'
+                            - 'MXN'
 
                 geographical:
                     options:
                         countries:
                             - 'US'
+                            - 'FR'
+                            - 'DE'
+                            - 'AU'
+                            - 'CA'
+                            - 'MX'
+                            - 'CK'
+                            - 'PT'
+                            - 'ES'
+                            - 'CN'
+                            - 'UK'
                         zones:
                             US:
                                 name: 'United States of America'
                                 countries:
                                     - 'US'
+                            WORLD:
+                                name: 'World'
```

```suggestion
                                name: 'Foreign countries'
```
or
```suggestion
                                name: 'Foreign zone'
```

I suppose someone may expect more countries in the world ;)

---

'World' is a value taken from task specification, so for now, I leave this name. but I think your suggestion is ok :)